lib/Rx/Observable/TimerObservable.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+namespace Rx\Observable;
+
+use Rx\Observable;
+use Rx\ObserverInterface;
+use Rx\SchedulerInterface;
+
+class TimerObservable extends Observable
```

Comment:
The other option would be:
```
timer($dueTime, $scheduler);
timerPeriodic($dueTime, $period, $scheduler)
```
And split the observable classes likewise.  

We can accomplish `timerPeriodic` with `interval` and `delay`, so it's probably not necessary.  We can always add it later, if we need to.